src/test/java/io/getunleash/util/UnleashConfigTest.java

```diff
@@ -128,8 +124,10 @@ public void should_add_app_name_and_instance_id_and_user_agent_to_connection()
         HttpURLConnection connection = (HttpURLConnection) someUrl.openConnection();
 
         UnleashConfig.setRequestProperties(connection, unleashConfig);
-        assertThat(connection.getRequestProperty(UNLEASH_APP_NAME_HEADER)).isEqualTo(appName);
-        assertThat(connection.getRequestProperty(UNLEASH_INSTANCE_ID_HEADER)).isEqualTo(instanceId);
+        assertThat(connection.getRequestProperty(UnleashConfig.UNLEASH_APP_NAME_HEADER))
+                .isEqualTo(appName);
+        assertThat(connection.getRequestProperty(UnleashConfig.UNLEASH_INSTANCE_ID_HEADER))
```

Comment:
Bring `UnleashConfig` static import back

Reply:
Guess IntelliJs refactor isn't perfect. Thanks for taking the trouble to point it out :)

Reply:
Fixed in 0906524